indico/core/plugins/__init__.py

```diff
@@ -34,12 +34,12 @@
 
 
 class PluginCategory(IndicoStrEnum):
-    search = _('Search')
+    search = pgettext('Plugin category title', 'Search')
     synchronization = _('Synchronization')
     payment = _('Payment')
     importers = _('Importers')
     videoconference = _('Videoconference')
-    other = _('Other')
+    other = pgettext('Plugin category title', 'Other')
 
```

Comment:
Wouldn't it make sense to give all of them the same context?

Reply:
It would, but adding context means we 'lose' the translation so I tried to add it only where necessary. I can add it though if you prefer